### PR TITLE
Changes to support populating multiple tokens in tokenField on selecting a single token in tableView

### PR DESCRIPTION
**Pull request by harshflock**
Changes for showing contact-lists in suggestions

### CODE REVIEW

File: KSTokenView/KSTokenView.swift [hunk 1]
```diff
@@ -69,11 +69,14 @@ import UIKit
    @objc optional func tokenViewDidBeginEditing(_ tokenView: KSTokenView)
    @objc optional func tokenViewDidEndEditing(_ tokenView: KSTokenView)
    
+   
    @objc func tokenView(_ tokenView: KSTokenView, performSearchWithString string: String, completion: ((_ results: Array<AnyObject>) -> Void)?)
    @objc func tokenView(_ tokenView: KSTokenView, displayTitleForObject object: AnyObject) -> String
    @objc optional func tokenView(_ tokenView: KSTokenView, withObject object: AnyObject, tableView: UITableView, cellForRowAtIndexPath indexPath: IndexPath) -> UITableViewCell
    @objc optional func tokenView(_ tokenView: KSTokenView, didSelectRowAtIndexPath indexPath: IndexPath)
-   
+   @objc optional func tokenView(_ tokenView: KSTokenView, didSelectRowObject object: AnyObject)
```
Comment (dineshflock) [inline KSTokenView/KSTokenView.swift]: didSelectRowAtIndexPath and didSelectRowObject, are almost same.
delegate has the array, and if I notify then index path selected or if I tell the object selected, its almost the same thing. 
So the delegate can get the selected row object inside the didSelectRowAtIndexPath call via `object = data[indexpath.row]`. So I see no reason to have another delegate method `didSelectRowObject `

Comment (dineshflock) [reply KSTokenView/KSTokenView.swift]: Either ways we should write a brief documentation above these methods as to when they would be called and what's expected to be returned, and what if nil is returned in `tokenObjectsFor` and things like this.

**Review by dineshflock — COMMENTED** [review]
@harsharora99

File: KSTokenView/KSTokenView.swift (hunk 1 above)
Comment (harshflock) [reply KSTokenView/KSTokenView.swift]: Hi @dineshflock 
For didSelectRowAtIndexPath, The array in which the objects are saved in KSTokenView (_resultArray) had fileprivate access so wasnt able to access that in delegate. Also, didnt found any function in KSTokenView to do so. Should I change the access specifier or add a function in KSTokeView to get access to its content?

Comment (harshflock) [reply KSTokenView/KSTokenView.swift]: Added description for tokenObjectsFor and didSelectRowObject delegate functions

**Pull request merged**